extensions/wikia/CreateNewWiki/classes/validation/RequestValidator.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Wikia\CreateNewWiki;
+
+use TorBlock;
+use WebRequest;
+
+class RequestValidator {
+	/**
+	 * @param WebRequest $request
+	 * @return bool
+	 * @throws MissingParamsException
+	 */
+	public function assertValidParams( WebRequest $request ): bool {
+		$params = $request->getArray( 'data' );
+
+		if (
+			empty( $params ) ||
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1067)